editions/tw5.com/tiddlers/howtos/Using Excise.tid

```diff
@@ -0,0 +1,22 @@
+created: 20160810122928198
+modified: 20160810122934291
+tags: Learning
+title: Using Excise
+type: text/vnd.tiddlywiki
+
+
+! Excise text
+From the EditorToolbar you can export selected text to a new tiddler and insert a [[link|Linking in WikiText]] [[Transclusion]] or [[macro|Macros]] in its place. Click ''Excise text'' ({{$:/core/images/excise}}), input name of the new tiddler, and choose excise method.
+
+! Excise a text
```

Comment:
Two times the same heading?  

I'd go with `! Excise text`  and may be `!! How to excise text` or just `!! HowTo`